shuup/importer/importing/importing.py

```diff
@@ -42,7 +44,7 @@ class DataImporter(object):
     def __init__(self, data, shop, language):
         self.shop = shop
         self.data = data
-        self.data_keys = data[0].keys()
+        self.data_keys = [k.replace(u'\ufeff', '') for k in data[0].keys()]
```

Comment:
Why don't you just remove the BOM from the beginning of the file while decoding the import file in the first place? It doesn't seem right to check BOMs at this point of import, since this phase should process already decoded data.